src/it/scala/io/iohk/ethereum/txExecTest/util/DumpChainApp.scala

```diff
@@ -73,10 +74,10 @@ object DumpChainApp extends App with NodeKeyBuilder with SecureRandomBuilder wit
     val peerMessageBus = actorSystem.actorOf(PeerEventBusActor.props)
 
     val peerManager = actorSystem.actorOf(PeerManagerActor.props(
+      peerDiscoveryManager = actorSystem.deadLetters, // TODO: fixme
```

Comment:
That's a good question. Do we still need DumpChainApp? If so, it would require to make it possible to specify a geth-like --nodiscovery flag.